control/config.py

```diff
@@ -157,7 +157,7 @@ class and functions.  If flat is `False`, then matrices are
     """
     if flag and warn:
         warnings.warn("Return type numpy.matrix is soon to be deprecated.",
-                      stacklevel=2)
+                      stacklevel=2, category=PendingDeprecationWarning)
```

Comment:
Hence the `PendingDeprecationWarning` instead of `DeprecationWarning` category. The warning message is already there, this is just adding the category.